jnigen/lib/src/summary/summary.dart

```diff
@@ -86,3 +91,85 @@ class SummarizerCommand {
     return proc;
   }
 }
+
+Future<Classes?> getSummary(Config config) async {
+  setLoggingLevel(config.logLevel);
```

Comment:
You're setting the logging level, here and in `generateJniBindings`.

Reply:
This is a potential entry point, eg during tests of this particular feature. If logs during that test is fine, I will remove this line.

In long term we may want to design a better logging abstraction. But I wouldn't worry about it a lot now.